package.json

```diff
@@ -173,7 +178,8 @@
    },
    "extensionDependencies": [
       "ms-kubernetes-tools.vscode-kubernetes-tools",
-      "ms-azuretools.vscode-docker"
+      "ms-azuretools.vscode-docker",
+      "vscode.git"
```

Comment:
💡 Thank you @davidgamero  for adding me to this review! Really appreciate it,  have we discussed what this dependency will lead to more towards, @qpetraroia question, user-scenario? 

Few questions I have in mind for understanding: 

Why: we have taken dependency on the `vscode:git` extension here which in itself is very cool and robust, so I am assuming we are planning to enable cross extension feature bridge of some kind here?

Question:

* Can you please share this here, or discussion or internally that do we have initial scenarios which will be the triggering point for this?

* Have we already thought about the way we want to bridge these commands? 

I was thinking if we just bridge the commands since we have created the extension dependency then we will not need the `api` transition, it will be as simple as - `user: you will leave this extension behaviour and now use the GitHub extension default behaviour` which I think will help us not to re-invent wheel and use their extension functionality?

Thanks heaps, cc: @rzhang628 and @gambtho ❤️🙏

Reply:
thanks Tats! the git extension is being used to list the branches on the current git repo, and it is a default extension for vscode so we don't require any extra effort from the users. We use this to allow the users to select which branch they want the workflow to deploy from in the quick pics.
Bridging the commands is an efficient way to work with other extensions, but I don't see an option to bridge for only selecting a branch and the handing  back control to our extension

Reply:
Nice! Thank you @davidgamero! another quick question, this `vscode:git` is this `scm` only dependency, I guess what I am trying understand for myself is the user-sceanario and if it is vscode `scm` then have we thought about `gh:copilot` route and something we can leverage from that?

Reply:
yes it's just for scm, Im not familiar with a way to get branch information from gh:copilot, but we could investigate that
i don't think copilot is a default extension, so less users may have it installed